main/core.js

```diff
@@ -130,10 +104,9 @@ async function start (core) {
     console.log(`Core closed all stdio with code ${code ?? '<no code>'}`)
 
     ;(async () => {
-      const log = await core.logs.get()
       Sentry.captureException('Core exited', scope => {
         // Sentry UI can't show the full 100 lines
-        scope.setExtra('logs', log.split('\n').slice(-10).join('\n'))
+        scope.setExtra('logs', logs.getLast(10))
```

Comment:
Before this change, you added last 10 lines of logs. Now you are adding last 10 chunks. I am not sure if chunks are always split on line boundaries. Maybe we should pipe `childProcess.stderr` through `split2` before we read the chunks into our `log` array?

Reply:
`Logs#push` performs splitting by newline, doesn't that make it work?

Reply:
What if the child process send a long line in two chunks? The first chunk does not contain any newline character, the second one does. Splitting on newlines will keep two lines in the `logs` array. Unless you join all items first and then split the combined string on newlines. (That feels very inefficient to me, TBH.)

Anyhow, not a big deal, I think this can produce an annoyance at most, it won't be a major problem.

Reply:
I see! The `split2` PR will help us make sure this is not an issue.